pkg/pillar/cmd/volumemgr/handlediskmetrics.go

```diff
@@ -132,9 +132,11 @@ func createOrUpdateDiskMetrics(ctx *volumemgrContext) {
 	log.Functionf("createOrUpdateDiskMetrics")
 	var diskMetricList []*types.DiskMetric
 	startPubTime := time.Now()
+	wdName := agentName + "metrics"
```

Comment:
This name could get out of synch with the caller so it would be better to have the caller pass wdName as an argument to createOrUpdateDiskMetrics.